src/utils/getCopyToClipboardContents.ts

```diff
@@ -8,7 +8,7 @@ import { Core } from "../tree/types";
 
 const NOT_DELETED = -1;
 
-const generateTreeCore = (treeCore: Core[]): Core[] => {
+export const generateTreeCore = (treeCore: Core[]): Core[] => {
   const visibleCoresAsLines: Core[] = [];
 
   treeCore.forEach((core) => {
```

Comment:
This is mutating the treeCore, try to DeepClone the object to stop modifying the existing object :)
Look you are doing core.treeCore = Index becuase FavIcon doens't get returned.
(The Function is Using the TreeCore Reference, you need to deep clone the object before use in the function on line 12.

Reply:
Oh!!!! I totally forgot that JavaScript makes a reference of variable and changing that reference would cause change of variable as well!!
Thanks I'll fix it

Reply:
I was going to mention this multiple times on our call but I thought theres no way this could be the bug but after further testing i was like lol it is...